test/rules.js

```diff
@@ -305,3 +305,58 @@ test( "rules(), rangelength attribute as array", function() {
 		rangelength: [ 2, 3 ]
 	} );
 } );
+
+test( "rules(), normalizer", function() {
+	$( "#usernamec" ).val( "\t\t \r" );
+	$( "#urlc" ).val( "" );
+
+	var v = $( "#testForm1clean" ).validate( {
+		rules: {
+			username: {
+				required: true,
+				// Using the normalizer to trim the value of the element
+				// before validating it.
+				normalizer: function( element ) {
```

Comment:
Wdyt of passing in the elementValue instead of the element? We could change `this` to be the element instead...